content/blog/federating-spire-on-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,452 @@
+---
+title: Federating SPIRE on HPE GreenLake for Private Cloud Enterprise
+date: 2023-05-15T03:37:14.807Z
+author: Akansha Sajimon, Nishant Chaturvedi
+authorimage: /img/Avatar1.svg
+disable: false
+tags:
```

Comment:
Please add one additional tag:
_spiffe-and-spire-projects_

This will allow to tie your blog post with the "SPIFFE and SPIRE Projects" landing page on HPE Developer Community portal: https://developer.hpe.com/platform/spiffe-and-spire-projects/home